openmetadata-ui/src/main/resources/ui/src/components/common/DomainLabel/DomainLabel.component.tsx

```diff
@@ -101,7 +102,7 @@ export const DomainLabel = ({
         <Typography.Text
           className={classNames(
             'domain-link',
-            !showDomainHeading ? 'font-medium text-xs' : ''
+            !showDomainHeading ? `font-medium text-xs ${className}` : ''
           )}
```

Comment:
Same here, if we don't want that to be applied, we should just not pass the className

Reply:
https://github.com/open-metadata/OpenMetadata/pull/15720#discussion_r1540896210